### PR TITLE
styling details page, second pass

### PR DESCRIPTION
**Pull request by jancborchardt**
Please review @DeepDiver1975 @Henni @owncloud/designers

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
I guess we should add some seperation between address and other fields

bildschirmfoto von 2016-03-02 16-49-01

File: templates/detailItems/adr.html [hunk 1]
```diff
@@ -1,17 +1,27 @@
 <select ng-model="ctrl.data.meta.type[0]">
 	<option ng-repeat="option in ctrl.availableOptions" value="{{option.id}}">{{option.name}}</option>
 </select>
-<label for="details-pobox">Post office box:</label>
-<input type="text" id="details-pobox" name="address" ng-model="ctrl.data.value[0]" value="" />
-<label for="details-address">Street address:</label>
-<input type="text" id="details-address" name="address" ng-model="ctrl.data.value[2]" value=""
+<!--<label for="details-pobox">Post office box:</label>
+<input type="text" id="details-pobox" name="address" ng-model="ctrl.data.value[0]" value="" />-->
```
Comment (Henni) [inline templates/detailItems/adr.html]: please remove the code comment

**Comment by Henni** [comment]
The arrow seems a bit of. On first sight one might wonder what it points to:
image
Maybe it would be better to combine the arrow with a square as it's done in other applications (e.g. [Wikipedia](https://upload.wikimedia.org/wikipedia/commons/6/64/Icon_External_Link.png))

Still broken: the telephone type
image

File: templates/detailItems/adr.html (hunk 1 above)
Comment (DeepDiver1975) [inline templates/detailItems/adr.html]: @jancborchardt we shall not remove the post office box - it's an valid part of the address

**Comment by DeepDiver1975** [comment]
possible fix for the select fields: http://stackoverflow.com/a/26627224

**Comment by DeepDiver1975** [comment]
rebased

**Comment by DeepDiver1975** [comment]
@jancborchardt @Henni I'm all in for merging in this state

**Comment by irgendwie** [comment]
Edit: Didn't completely read @Henni comment... LGTM then :+1:

Is this expected behavior for telephone numbers?

phone

**Comment by Henni** [comment]
Better than before :+1:

**Comment by DeepDiver1975** [comment]
rebased again :+1:

**Pull request merged**